Assets/Shopify/UIToolkit/Shops/Generic/Classes/GenericCartView.cs

```diff
@@ -0,0 +1,52 @@
+namespace Shopify.UIToolkit.Shops.Generic {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using System.Collections.Generic;
+    using System;
+
+    public class GenericCartView : GenericMultiProductShopView {
```

Comment:
Should this just be named CartView? (The other views don't seem to have this prefix..?)

Reply:
I'm not sure because the interface we implement is named `GenericMultiProductShopView`. I feel like if the interface is prefixed with `Generic` then it make sense for this to be `Generic` as well?

Reply:
Fair enough, but I think that would imply we rename all the other views as well, perhaps in another PR?

Reply:
Ah I see what you mean - like CartItemView and stuff. Yea I wonder if we want to either prefix all this with Generic? Or if having them in the folder is enough for people to know?

Reply:
imho adding generic in front doesn't add anything. I suspect people will just copy these classes, or the prefabs wholesale, so these are probably more likely to _be_ the the implementation in the end.